### PR TITLE
RHEL-8: Migrate `makebumpver` script from Bugzilla to JIRA

### PR DESCRIPTION
**Pull request by jkonecny12**
Migrate our current `makebumpver` script from Bugzilla checking to JIRA.

We are changing the convention from:
```
Resolves: rhbz#<bug number>
Related: rhbz#<bug number>
```
to
```
Resolves: <RHEL issue>
Related: <RHEL issue>
Reverts: <RHEL issue>
```
These scripts are a bit outdated so let's take a brief change on this during the path:

* Base migration to pep8 (still names of the variables and functions are not correct)
* Backport fix to work with `infra: ...` together with `... (#infra)` in the infra commits - improves backporting for us
* Improve duplication in the code
* Improve logging - less mess up output compared until you enable --debugging
* Switch from `sys.std***` to `print`
* Backporting fixes to the templating system
* Fixing up the documentation
* Removing outdated git scripts

This PR is bacport of:
* https://github.com/rhinstaller/anaconda/pull/5351
* https://github.com/rhinstaller/anaconda/pull/5348
* https://github.com/rhinstaller/anaconda/pull/5349
* https://github.com/rhinstaller/anaconda/pull/5366
* https://github.com/rhinstaller/anaconda/pull/5364

### CODE REVIEW

**Comment by jkonecny12** [comment]
Blocked until all the PRs backported here are merged.

**Review by jstodola — APPROVED** [review]
It looks good to me.

**Review by VladimirSlavik — APPROVED** [review]

**Pull request merged**
